lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -54,25 +55,58 @@ function formatNumber(number) {
   return number.toLocaleString(undefined, {maximumFractionDigits: 1});
 }
 
+/**
+ * Format time.
+ * @param {string} date
+ * @return {string}
+ */
+function formatDateTime(date) {
+  const options = {
+    month: 'numeric', day: 'numeric', year: 'numeric',
+    hour: 'numeric', minute: 'numeric', timeZoneName: 'short'
+  };
+  let formatter = new Intl.DateTimeFormat('en-US', options);
```

Comment:
The devtools mocks make this even nicer:

![screen shot 2017-04-13 at 1 43 47 pm](https://cloud.githubusercontent.com/assets/238208/25023609/579c0626-204f-11e7-9b9e-9ade2d348258.png)

What if we go with that?